src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -36,6 +36,8 @@ public class MainWindow extends UiPart<Stage> {
     private HelpWindow helpWindow;
     private StatisticsWindow statisticsWindow;
 
```

Comment:
Empty line between StatisticsWindow and ScheduleListPanel

Reply:
Sure, will fix it